scripts/CMakeLists.txt

```diff
@@ -1,11 +0,0 @@
-cmake_minimum_required(VERSION 3.6)
```

Comment:
Now that conf is no longer a dependency we need to update the mac documentation that instructs users to build conf.

Reply:
Nice catch. @ulfalizer can you get rid of the paragraph beginning with "Build Kconfig in :file:`$ZEPHYR_BASE/build` and add it to path" in  https://github.com/zephyrproject-rtos/zephyr/blob/master/doc/getting_started/installation_mac.rst

Reply:
Same for the paragraph beginning with "Within the MSYS console, build Kconfig in :file:`$ZEPHYR_BASE/build` and add it to path" in https://github.com/zephyrproject-rtos/zephyr/blob/master/doc/getting_started/installation_win.rst

Reply:
And same with the paragraph beginning with "On UNIX platforms, Build Kconfig in :file:`$ZEPHYR_BASE/build` and add it to path" in https://github.com/zephyrproject-rtos/zephyr/blob/master/doc/getting_started/getting_started.rst. Also remove "Now that the host tools are installed,"

Reply:
Fixed